org.eclipse.january.test/src/org/eclipse/january/dataset/LazyMathsTest.java

```diff
@@ -14,63 +14,60 @@
 import org.eclipse.january.dataset.DatasetFactory;
 import org.eclipse.january.dataset.LazyMaths;
 import org.eclipse.january.dataset.Maths;
-import org.junit.BeforeClass;
+import org.junit.Before;
 import org.junit.Test;
 
 public class LazyMathsTest {
 
-	@BeforeClass
-	public static void init() {
-		// ensure we will not be using the faster Dataset methods!
-		LazyMaths.setAllowDatasetMaths(false);
+	private ILazyDataset a2x2L;
+	private ILazyDataset b4x4x4x4L;
+	private ILazyDataset c4x5x6x7L;
+	private Dataset a2x2D;
+	private Dataset b4x4x4x4D;
+	
+	@Before
+	public void init() throws Exception {
+		a2x2L = LazyDataset.createLazyDataset(DatasetFactory.createRange(4).reshape(2, 2));
```

Comment:
Is it more logical to create the `Dataset`s first then use the utility wrapper method to create `LazyDataset`s?